distributor-api-v1/operations.md

```diff
@@ -123,48 +122,149 @@ Preferred initial call used to obtain all static data about distributor configur
+| `Theme` | [Theme](operations.md#theme) | optional | Distributor's theme variant. |
+| `VoucherCode` | string | optional | Voucher code which enables special rate offerings. |
+
+#### Theme
+
+* `Light`
+* `Dark`
+
+#### City   
+
+|  | Property | Type | Description |
+| :--- | :--- | :--- | :--- |
+| `Id` | string | required | Unique identifier of the city. |
+| `ImageId` | string | optional | Unique identifier of the city image. |
+| `Name` | string | [LocalizedText](operations.md#localizedtext) | City name. |
+
+#### Configuration
+
+|  | Property | Type | Description |
+| :--- | :--- | :--- | :--- |
+| `Id` | string | required | Unique identifier of the configuration. |
+| `AdultCount` | number | optional | Default number of adults. |
+| `ChildCount` | number | optional | Default number of children. |
+| `ChildSelectionEnabled` | boolean | optional | Determines whether to allow adding children to reservations (true by default). |
+| `CompetitorPriceRelativeAdjustment` | number | optional | Percentage markup with which competitor's prices (listed in the rate comparison banner if `DisplayRateComparison` is set to `true`) will be shown, compared to enterprise's Best Available Rate (BAR). For example, if enterprise's BAR costs 50, and entered here is "100", their rate will be shown as 50. If here is entered "110", their rate will be shown as 55 (as here is added a 10% markup). |
+| `CompetitorRateDescription` | [LocalizedText](operations.md#localizedtext) | required | Description differentiating enterprise's online booking from competitors booking. (for example, `20% online booking discount` or `Breakfast included`). |
+| `DisplayAvailability` | boolean | optional | Determines whether to display property's availability next to maximum occupancy in space categories (availability will be shown by default). |
+| `DisplayRateComparison` | boolean | optional | Determines whether to display rate comparison. |
+| `DisplaySpecialRequests` | boolean | optional | Determines whether to display special requests field during checkout. |
+| `Enterprise` | [Enterprise](operation.md#enterprise) | required | Enterprise to which the `Configuration` belongs. |
+| `OnlineTravelAgencies` | array of string | required | List of travel agencies to include in comparison banner. |
+| `PaymentCardInput` | string [PaymentCardInput](operations.md#paymentcardinput) | required | Determines how to handle payment cards. |
+| `RequiredFields` | array of [RequiredField](operations.md#requiredfield) | required | Form fields which are required and need to be filled in. |
+| `ServiceId` | string | required | Unique identifier of the service to which the configuration is bound to. |
+
+#### PaymentCardInput
+
+* `NotRequested` - Payment card info is not requested.
+* `Requested` - Payment card info is requested, but not validated.
+* `Required` - Payment card info is requested and validated. 
+
+#### RequiredField
+
+* `Telephone`
```

Comment:
There is really only one?

Reply:
Yes:
![image](https://user-images.githubusercontent.com/6526378/79575824-a779a780-80c2-11ea-9202-c25b90dd5dd8.png)

Reply:
for now, it might get extended in future